Firebase/Auth/Source/FIRAuth.m

```diff
@@ -376,14 +386,21 @@ - (instancetype)initWithAPIKey:(NSString *)APIKey appName:(NSString *)appName {
       FIRLogError(kFIRLoggerAuth, @"I-AUT000001",
                   @"Error loading saved user when starting up: %@", error);
     }
+
+    #if TARGET_OS_IOS
     // Initialize for phone number auth.
     _tokenManager =
         [[FIRAuthAPNSTokenManager alloc] initWithApplication:[UIApplication sharedApplication]];
+    #endif
+
     _appCredentialManager = [[FIRAuthAppCredentialManager alloc] initWithKeychain:_keychain];
```

Comment:
Please include all references to _appCredentialManager to the inside of `#if TARGET_OS_IOS`.